api/requirements.txt

```diff
@@ -3,7 +3,7 @@ flask
 flask_paginate
 lxml
 rdflib
-pyldapi
+pyldapi==2.1.3

```

Comment:
Change to pyldapi<3.0
Theres a chance we could release more versions in the 2.x.x series.

Reply:
good suggestion. done.